README-saml.md

```diff
@@ -43,4 +44,38 @@ Various URLs that you will need:
     cat myservice.key | base64 -w0
     ```
     Add this string to the `platform` section of your Terraform configuration as the `saml_key` variable. If you are using the encrypted blob, feel free to add this information to that configuration instead: `saml_key={base64 encoded key}`.
+
 1. Run a `terraform apply` and browse to your `{endpoint}` URL. You should be redirected to your IDP's login screen to begin the authentication process. If you are not redirected and instead are dropped into the application, you may still be logged in as the `admin@graymeta.com` account. If that is the case, click the logout button and you should get redirected to your IDP login screen.
+
+### Azure AD Procedure
+
+1. Create an Enterprise App using the '	Integrate any other application you don't find in the gallery' option, then proceed to configure SSO/SAML.
+
+1. Set the Identifier (Entity ID) as `{endpoint}/saml/acs`
+
+1. Set the Reply URL (Assertion Consumer Service URL) as `{endpoint}/saml/metadata`
+
+1. Configure 'Single Sign-On with SAML' in the Azure portal with the following attribute statements, and set your variables in Terraform:
```

Comment:
Maybe backtick the 'Single Sign-On with SAML'

Reply:
fixed